Sources/Basic/Radius.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+import UIKit
+
+/// An enumeration to describe differnt types of `Radius`.
```

Comment:
I think `Radius` would be better called `BorderRadius`: it is definitely tied to the specific use-case of view-borders because of the `circular` case, which doesn't make sense in a general setting. `Radius` seems to be a too general name here.

Optionally, and maybe more sensible, `Radius` could be a subtype of `BorderStyle`.

Then you could also change the doc comment to: `An enumeration describing different kinds of border radii`.